sdk/confidentialledger/azure-security-confidentialledger/src/test/java/com/azure/security/confidentialledger/ConfidentialLedgerIdentityClientTestBase.java

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.security.confidentialledger;
+
+import com.azure.core.test.TestBase;
+
+class ConfidentialLedgerIdentityClientTestBase extends TestBase {
```

Comment:
This class isn't used anywhere (from what I can see). Do we need this?

Reply:
Since this was autogenerated, I believe it would be best to leave it in. Is that reasonable?